### ISSUE TITLE
Crash when Viewing participant using Jisti Android SDK 5.0.2

### ISSUE THREAD

**Issue by bmarty** [jitsi/jitsi-meet#11375]
### Steps to reproduce:

1. Start a Jitsi call and join the conf
2. Click on the three dot menu and on "Participants"

The app crashes.

### Expected behavior:

The app does not crash and we can see the participants

### Actual behavior:

The app crashes

### Server information:

- Jitsi Meet version: 5.0.2
- Operating System: Android

### Client information:

- Browser / app version:
- Operating System:

### Additional information:

Stack trace:

```
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean com.facebook.react.uimanager.FabricViewStateManager.hasStateWrapper()' on a null object reference
        at com.facebook.react.views.textinput.ReactEditText.updateCachedSpannable(ReactEditText.java:981)
        at com.facebook.react.views.textinput.ReactEditText.access$300(ReactEditText.java:75)
        at com.facebook.react.views.textinput.ReactEditText$TextWatcherDelegator.onTextChanged(ReactEditText.java:1092)
        at android.widget.TextView.sendOnTextChanged(TextView.java:11631)
```

Originally posted here: https://github.com/vector-im/element-android/issues/5745

**Comment by bmarty** [comment]
OK, thanks for the feedback. We will wait for the SDK update, since we observe only a few occurrences of this crashes on our side.

**Comment by saghul** [comment]
This has been fixed in SDK 5.1.X.

**Comment by bmarty** [comment]
Yes, I am currently upgrading our dep to 5.1.0. Will confirm if this is fixed.

**Comment by bmarty** [comment]
OK, I confirm the issue is fixed with Jitsi 5.1.0!

<img width="292" alt="image" src="https://user-images.githubusercontent.com/3940906/171050539-3c9a7548-510b-47ef-ba1d-311396c826ac.png">
